food.py

```diff
@@ -0,0 +1,20 @@
+from turtle import Turtle
+import random
+
+
+class Food(Turtle):
+
+    def __init__(self):
+        super().__init__()
```

Comment:
Looks fine but my biggest critique is that I like to stash constants at the top of the page so that they're not buried in functions.

Reply:
included json for constants, is this what you meant?